partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnDivide.kt

```diff
@@ -21,7 +21,7 @@ internal object FnDivide : DiadicArithmeticOperator("divide") {
             if (arg1 == 0.toByte()) {
                 throw DataException("Division by zero for TINYINT: $arg0 / $arg1")
             } else if (arg0 == Byte.MIN_VALUE && arg1.toInt() == -1) {
-                throw DataException("Overflow for TINYINT: $arg0 / $arg1")
+                throw DataException("Resulting value out of range for: $arg0 / $arg1")
```

Comment:
For the following 3, you removed the type name.

Reply:
Ah my bad. I will fix in the target branch.